### ISSUE TITLE
last shub version seems broken

### ISSUE THREAD

**Issue by kalessin** [scrapinghub/shub#283]
This is happening now with last version of shub. I avoid the problem by pining to shub==2.5.1
```
Traceback (most recent call last):
  File "/home/ubuntu/virtualenv/python2.7/bin/shub", line 11, in <module>
    sys.exit(cli())
  File "/home/ubuntu/virtualenv/python2.7/local/lib/python2.7/site-packages/click/core.py", line 722, in __call__
    return self.main(*args, **kwargs)
  File "/home/ubuntu/virtualenv/python2.7/local/lib/python2.7/site-packages/click/core.py", line 697, in main
    rv = self.invoke(ctx)
  File "/home/ubuntu/virtualenv/python2.7/local/lib/python2.7/site-packages/click/core.py", line 1066, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/home/ubuntu/virtualenv/python2.7/local/lib/python2.7/site-packages/click/core.py", line 895, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/home/ubuntu/virtualenv/python2.7/local/lib/python2.7/site-packages/click/core.py", line 535, in invoke
    return callback(*args, **kwargs)
  File "/home/ubuntu/virtualenv/python2.7/local/lib/python2.7/site-packages/shub/deploy.py", line 79, in cli
    conf=conf)
  File "/home/ubuntu/virtualenv/python2.7/local/lib/python2.7/site-packages/shub/deploy.py", line 107, in deploy_cmd
    egg, tmpdir = _build_egg()
  File "/home/ubuntu/virtualenv/python2.7/local/lib/python2.7/site-packages/shub/deploy.py", line 161, in _build_egg
    settings = get_config().get('settings', 'default')
  File "/usr/lib/python2.7/ConfigParser.py", line 607, in get
    raise NoSectionError(section)
ConfigParser.NoSectionError: No section: 'settings'
```

**Comment by kalessin** [comment]
@jdemaeyer

**Comment by jdemaeyer** [comment]
Thanks for the report martin, fixed in #284. I'll make a release today
